src/v/cluster/controller_api.cc

```diff
@@ -74,16 +74,17 @@ controller_api::all_reconciliations_done(std::vector<model::ntp> ntps) {
         co_await ss::parallel_for_each(
           ntps.begin() + i,
           ntps.begin() + i + this_batch,
-          [this, reduced](const model::ntp& ntp) -> ss::future<> {
```

Comment:
i guess this would be a case where ss::lambda() is applicable since the overall statement is also being co_awaited. but i wouldn't change it now.